### PR TITLE
nrepl: support pprinting eval results

### PR DESCRIPTION
**Pull request by maxrothman**
Fixes https://github.com/BetterThanTomorrow/joyride/issues/49

### CODE REVIEW

**Review by PEZ — CHANGES_REQUESTED** [review]
Thanks for this PR! Changes LGTM.

Please also add an entry under `[Unreleased]` in CHANGELOG.md. Make it a link to the issue you are solving.

Note to self: Create a PR template.

**Comment by borkdude** [comment]
I will test a similar PR in nbb which has proper tests for this (which we don't have here yet). Maybe wait with merging until that PR has been merged?

**Comment by borkdude** [comment]
After fixing a test, the nbb PR was merged.

**Comment by PEZ** [comment]
Then I'm only waiting for @maxrothman to update the Changelog here 😄

File: src/main/joyride/nrepl.cljs [hunk 1]
```diff
@@ -65,6 +66,23 @@
             "ops" (zipmap (map name (keys ops)) (repeat {}))
             "status" ["done"]}))
 
+(def pretty-print-fns-map
+  {"clojure.core/prn" prn
+   "clojure.pprint/pprint" pp/pprint
```
Comment (phrohdoh) [inline src/main/joyride/nrepl.cljs:71]: Being `.cljs` I find myself using `cljs.pprint/pprint` often, could we ~~sneak that in~~ add that here as well?

**Review by PEZ — APPROVED** [review]

File: src/main/joyride/nrepl.cljs (hunk 1 above)
Comment (borkdude) [reply src/main/joyride/nrepl.cljs:71]: Yes

Comment (PEZ) [reply src/main/joyride/nrepl.cljs:71]: Help me understand. Does the `clojure.pprint` namespace get available to joyride scripts from this? Testing the VSIX and trying to require it, I get:

```
; ENOENT: no such file or directory, open '/Users/pez/.config/joyride/scripts/.joyride/scripts/clojure/pprint.cljs'
```

Comment (borkdude) [reply src/main/joyride/nrepl.cljs:71]: No it doesn't. We need to add that in the SCI config. But clojure.pprint does get used here to print values sent back the the nREPL client.

Comment (PEZ) [reply src/main/joyride/nrepl.cljs:71]: So then the addition of `cljs.pprint` is for someone who configures the nrepl printer?

Comment (borkdude) [reply src/main/joyride/nrepl.cljs:71]: Yes, but by default CIDER uses a cider-specific function. We just map that to cljs.pprint.

**Pull request merged**